### PR TITLE
Pokedex component

### PR DESCRIPTION
**Pull request by nathanSamosa**

### CODE REVIEW

File: src/client/App.css [hunk 1]
```diff
@@ -24,8 +24,13 @@
...
     --clr-type-ice: 80, 195, 230;
     --clr-type-ghost: 123, 98, 183;
-
     --clr-type-flying: 238, 237, 239;
     --clr-type-ground: 214, 198, 126;
     --clr-type-dragon: 241, 110, 87;
+
+
+    --clr-star-filled: 255, 193, 7;
+    --clr-star-hollow: 228, 229, 223;
```
Comment (Can-Tina) [inline src/client/App.css:33]: More variables, nice

File: src/client/styling/pokedex.css [hunk 2]
```diff
@@ -26,8 +26,8 @@
     text-align: center;
     padding: 0.5rem;
     grid-template-columns: 1fr;
-    box-shadow: 0px 0px 10px rgb(var(--clr-shadow));
-
+    box-shadow: -2px -2px 5px rgba(var(--clr-pokeblue), 0.5), 
+                2px 2px 5px rgba(var(--clr-pokeorange), 0.5)
```
Comment (Can-Tina) [inline src/client/styling/pokedex.css:30]: Pretty :)

File: src/server/controllers/user.js [hunk 3]
```diff
@@ -33,15 +33,17 @@ const postRegisterDetails = async (req, res) => {
...
         const { username, password } = req.body;
         const findUser = await prisma.user.findUnique({
             where: {
                 username
             }
         });
+        console.log(findUser)
 
-        const matchingPassword = await bcrypt.compare(password, findUser.password);
+        const matchingPassword = findUser ? await bcrypt.compare(password, findUser.password) : null;
```
Comment (Can-Tina) [inline src/server/controllers/user.js:46]: You like this format haha

Comment (nathanSamosa) [reply src/server/controllers/user.js:46]: lol i sure do. We need this to return either truthy or falsy; this seems to be the best way to do it as using an IF statement means declaring `matchingPassword` outside of the statement first

File: src/client/styling/pokedex.css (hunk 2 above)
Comment (glowkeeper) [reply src/client/styling/pokedex.css:30]: Oooo - css variables - nice :)

File: src/server/controllers/user.js (hunk 3 above)
Comment (glowkeeper) [reply src/server/controllers/user.js:46]: Nathan and his ternaries :)

**Comment by glowkeeper** [comment]
Approved

File: src/client/styling/pokedex.css (hunk 2 above)
Comment (nathanSamosa) [reply src/client/styling/pokedex.css:30]: > Oooo - css variables - nice :)

If you like those, wait till you see my wordle clone ;)

Comment (nathanSamosa) [reply src/client/styling/pokedex.css:30]: @glowkeeper If you like those, wait till you see my wordle clone ;)

**Pull request merged**